initial draft of 2022-03-20 release notes (#2863)
Azure/AKS: CHANGELOG.md

# Azure Kubernetes Service Changelog

## Release 2022-03-20

This release is rolling out to all regions - estimated time for completed roll out is 2022-04-03 for public cloud and 2022-04-06 for sovereign clouds. Please note that the AKS release cadence has shifted; new releases will now be cut on Sunday.

### Announcements

* Upgrade your AKS Ubuntu 18.04 worker nodes to VHD version 2022.03.20 or newer to address [CVE-2022-0492](https://github.com/Azure/AKS/issues/2834) and [CVE-2022-23648](https://github.com/Azure/AKS/issues/2821).

> * The [single placement group VMSS flag](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-hpc#:~:text=Virtual%20machine%20scale,increased%20to%20300.) will now be enabled for newly-created node pools using InfiniBand/RDMA-capable VM sizes. InfiniBand/RDMA-capable SKUs, like most [H-series](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-hpc) and some [N-series](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu) sizes, can be identified by the "r" in the additional features section of the size name (e.g. Standard_HB120**r**s_v3, Standard_ND96as**r**_v4). Note that the InfiniBand drivers are not currently loaded to nodes. Loading these via a DaemonSet will come in the near future.

Reviewer: this is a behavior change or a feature. Not an announcement.

Author: Moved.

> * The [single placement group VMSS flag](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-hpc#:~:text=Virtual%20machine%20scale,increased%20to%20300.) will now be enabled for newly-created node pools using InfiniBand/RDMA-capable VM sizes. InfiniBand/RDMA-capable SKUs, like most [H-series](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-hpc) and some [N-series](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu) sizes, can be identified by the "r" in the additional features section of the size name (e.g. Standard_HB120**r**s_v3, Standard_ND96as**r**_v4). Note that the InfiniBand drivers are not currently loaded to nodes. Loading these via a DaemonSet will come in the near future.

Author: Fixed, it was meant to point to a specific section but that's not multi-language friendly. Updated it to a tag instead.

> * Accelerated networking will now be enabled by default for newly-created Windows nodepools.

Reviewer: this is a behavior change or a feature. Not an announcement.

Author: Moved

> * [Calico network policies for Windows nodes](https://docs.microsoft.com/en-us/azure/aks/use-network-policies) are now GA.

Reviewer: this is a feature. Not an announcement.

Author: Moved

> * [Node pool snapshot](https://docs.microsoft.com/en-us/azure/aks/node-pool-snapshot) is now GA.

Reviewer: Feature

Author: Moved

> * The 2022.03.20+ AKS Ubuntu 18.04 images fix an issue (present since 2022.02.19) in which an unexpected security agent to be installed, leading to higher than expected memory consumption on nodes.

Reviewer 2: nit: awkward words?

> * The 2022.03.20+ AKS Ubuntu 18.04 images fix an issue (present since 2022.02.19) in which an unexpected security agent to be installed, leading to higher than expected memory consumption on nodes.

Author: ```suggestion
* The 2022.03.20+ AKS Ubuntu 18.04 images fix an issue (present since 2022.02.19) in which an unneeded Azure security agent was installed, leading to higher than expected memory consumption on nodes.
```

> * The 2022.03.20+ AKS Ubuntu 18.04 images fix an issue (present since 2022.02.19) in which an unneeded Azure security agent was installed, leading to higher than expected memory consumption on nodes.

Reviewer: Bugfix

Author: Moved

> * From Kubernetes 1.23, containerD will be the default container runtime for Windows node pools. Docker support will be deprecated in Kubernetes 1.24. You are advised to test your workloads before Docker deprecation happens by following the documentation [here](https://docs.microsoft.com/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview).

Reviewer 2: nit: I don't think containerd stylize their name that way
